client/src/utils/queryDuels.js

```diff
@@ -50,7 +67,7 @@ export function makeGetDuelDetails(duelId) {
     // TODO > the request duel request body to prevent request chaining
```

Comment:
While we are here, can we remove this comment and replace it with an Issue? The idea here is to put more informations into the duel request (like the user avatars) because we will need them anyway

Reply:
Yes, I let you create this issue !